src/main/edit_delete/Clip.ts

```diff
@@ -0,0 +1,16 @@
+export default class Clip {
```

Comment:
Is this class going to have any specific methods attached to it? If not, can it just be an object with an interface instead? (better for logging, passing around, serialising to a string, etc)

Reply:
I get what you mean - Clip doesn't really need to be a class. Sorry but I'm not too sure how to implement the change though, my JS and TS is worse than my Japanese, and I don't speak Japanese.

Reply:
Hahah, that's unfortunate - my Japanese isn't too good either 😢

Basically just instead of calling `new Clip`, create an object directly with the fields it needs - e.g. `{ startTime: 123, duration: 456, fileName: 'bla' }` and that should be the only change you need (accessing fields is the same as with a class)

If you can't get it working, don't worry too much, it's not a blocker.

Reply:
Oh, should probably add, specify the type of the object as `Clip` and then make an interface for it:

```
interface Clip {
    startTime: number;
    duration: number;
    fileName: string;
}
```